packages/seacas/libraries/ioss/src/exodus/Ioex_BaseDatabaseIO.C

```diff
@@ -507,9 +507,33 @@ namespace Ioex {
     }
 
     ex_set_max_name_length(m_exodusFilePtr, maximumNameLength);
+
+    open_root_group_nl();
+    open_child_group_nl(0);
+  }
+
+  bool BaseDatabaseIO::supports_internal_change_set_nl()
+  {
+    return supports_group_nl();
   }
 
-  bool BaseDatabaseIO::open_root_group_nl()
+  bool BaseDatabaseIO::supports_group_nl() const
```

Comment:
Does this need a lock/no-lock version.  There should be no way tha another thread could change the format of this file while it is being queried.

(I probably need to get back in the thread-safe zen mode more for 100% confidence...)

Reply:
Yes, that should be good.  If they make changes down at the exodus level, then exodus should take care of locking, so should only have to look at what happens at the IOSS level.